boards/arm/xmc45_relax_kit/Kconfig.defconfig

```diff
@@ -8,4 +8,15 @@ if BOARD_XMC45_RELAX_KIT
 config BOARD
 	default "xmc45_relax_kit"
 
+if NETWORKING
+
+config NET_L2_ETHERNET
+	default y
+config MDIO
+	default y
+config TEST_RANDOM_GENERATOR
```

Comment:
This seems to be selected by many boards but just wondering isn't there a proper rng we could use here?
The config option specifically says that this option should not be selected by the board
```
          This option is intended to be selected only by application-level
          configurations (e.g. in tests and samples) to indicate that the
          application is allowed to run with a random number generator that is not
          truly random. Board-level configurations must not select this option
          unless the sole purpose of the board is testing (e.g. QEMU emulation
          boards).
```

Reply:
On the XMC4xxx there is a PRNG in the DAC module, but there's no dedicated PRNG.
Also I don't know how much more random this PRNG would be than the method implemented in subsys/random/rand32_timer.c

I'll do a bit more reading.

![image](https://github.com/zephyrproject-rtos/zephyr/assets/17433307/0616c2c4-7c44-4a9b-b782-d82e3470fdb5)